src/components/Loader/index.tsx

```diff
@@ -7,13 +7,21 @@ interface LoaderProps {
   delay?: number;
   message?: string;
   showLoading: boolean;
+  backdropDismiss?: boolean;
 }
 
-const Loader: React.FC<LoaderProps> = ({ delay, message, showLoading }) => {
+const Loader: React.FC<LoaderProps> = ({ backdropDismiss = false, delay, message, showLoading }) => {
   return useDelayedRender(
     delay ?? 500,
     showLoading
-  )(() => <IonLoading isOpen={showLoading} message={message || 'Please wait...'} spinner="lines" />);
+  )(() => (
+    <IonLoading
+      isOpen={showLoading}
+      message={message || 'Please wait...'}
```

Comment:
Discovering TDEX providers...

Reply:
'Please wait...' is the default message of generic Loader component.
The message is 'Discovering TDEX providers with best liquidity...'